speechAssets/TOPIC.txt

```diff
@@ -1,10 +1,16 @@
 UK
+uk
+u. k.
```

Comment:
remember that any time you make a change to the topics, you have to make a corresponding change in src/capiQueryBuilder.js :(
I've just pushed a commit to fix this